system/component.cue

```diff
@@ -8,28 +8,29 @@ component: schema.#component & {
 	handle: "system"
 
 	resources: {
-		"attachment":            attachment
-		"application":           application
-		"apigw-route":           apigw_route
-		"apigw-filter":          apigw_filter
-		"auth-client":           auth_client
-		"auth-confirmed-client": auth_confirmed_client
-		"auth-session":          auth_session
-		"auth-oa2token":         auth_oa2token
-		"credential":            credential
-		"data-privacy-request":  data_privacy_request
-		"queue":                 queue
-		"queue_message":         queue_message
-		"reminder":              reminder
-		"report":                report
-		"resource-translation":  resource_translation
-		"role":                  role
-		"role_member":           role_member
-		"settings":              settings
-		"template":              template
-		"user":                  user
-		"dal_connection":        dal_connection
-		"dal_sensitivity_level": dal_sensitivity_level
+		"attachment":            					attachment
+		"application":           					application
+		"apigw-route":           					apigw_route
+		"apigw-filter":          					apigw_filter
+		"auth-client":           					auth_client
+		"auth-confirmed-client": 					auth_confirmed_client
+		"auth-session":          					auth_session
+		"auth-oa2token":         					auth_oa2token
+		"credential":            					credential
+		"data-privacy-request":  					data_privacy_request
+		"data-privacy-request_comment":  	data_privacy_request_comment
+		"queue":                 					queue
+		"queue_message":         					queue_message
+		"reminder":              					reminder
+		"report":                					report
+		"resource-translation":  					resource_translation
+		"role":                  					role
+		"role_member":           					role_member
+		"settings":              					settings
+		"template":              					template
+		"user":                  					user
+		"dal_connection":        					dal_connection
+		"dal_sensitivity_level": 					dal_sensitivity_level
```

Comment:
Can you fix up the indentation here?

Reply:
It's fixed I guess, It's just an ident issue here.